src/cassandra/statement.rs

```diff
@@ -283,6 +285,14 @@ impl Statement {
         }
     }
 
+    /// Sets the statement's timeout for waiting for a response from a node.
```

Comment:
This should return `&mut Self`, not just `&Self`, so other modifiers can be chained.

Reply:
Thanks